src/frontend/screens/ObservationsList/ObservationListItem.tsx

```diff
@@ -84,14 +85,16 @@ const ObservationListItem = ({
         {photos.length ? (
           <View style={styles.photoContainer}>
             <PhotoStack photos={photos} />
-            <CategoryCircleIcon
-              iconId={iconId}
-              size="small"
-              style={styles.smallIcon}
-            />
+            <View style={styles.smallIconContainer}>
+              <CategoryCircleIcon
+                iconId={iconId}
+                color={iconColor}
+                size="small"
+              />
+            </View>
```

Comment:
More of a refactor change, but this was the only place that used the `style` override prop for `CategoryCircleIcon` (note, _not_ `Circle`) and it only adjusts the positioning of the icon, so decided to wrap it in order to remove that prop altogether